com.qzandronum.Q-Zandronum.yaml

```diff
@@ -0,0 +1,163 @@
+app-id: com.qzandronum.Q-Zandronum
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: 5.15-22.08
+command: doomseeker.sh
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+
+  # Based on GZDoom
+  # Now for something confusing: Doomseeker honours XDG standards
+  # But Zandronum doesn't. Both persist for consistency.
+  - --env=DOOMWADDIR=/app/share/games/doom
+  - --persist=.config/zandronum
+  - --persist=.config/doomseeker
+
+cleanup:
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /lib/pkgconfig
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/glew/glew.json
+  - shared-modules/gtk2/gtk2.json
+
+  - shared-modules/SDL2/SDL2-no-libdecor.json
+  - shared-modules/SDL/sdl12-compat.json
+
+  # with libjpeg.so.8
+  - name: libjpeg
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_SKIP_RPATH:BOOL=YES
+      - -DENABLE_STATIC:BOOL=NO
+      - -DWITH_JPEG8:BOOL=YES
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib
+    sources:
+      - type: archive
+        url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/refs/tags/2.1.3.tar.gz
+        sha256: dbda0c685942aa3ea908496592491e5ec8160d2cf1ec9d5fd5470e50768e7859
+
+  - name: openssl11
```

Comment:
openssl 1.1 compatibility is supposed to be available in the runtime. doesn't that work?

Reply:
It does, thanks for noticing.